apps/gallery/.storybook/main.ts

```diff
@@ -1,7 +1,7 @@
 /** @type { import('@storybook/web-components-vite').StorybookConfig } */
 const config = {
   stories: ['../stories/**/*.mdx', '../stories/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials', 'storybook-dark-mode'],
```

Comment:
Instead of using an adon from a third-party, can we utalise default storybook packages / manager api for this? https://storybook.js.org/docs/web-components/configure/theming

Reply:
I removed the dark mode addon. Dark mode is enabled by default and we can switch theme in the toolbar above. Would be nice if we can hook the `ThemeUtil` to this switch in the future.